released/packages/coq-mathcomp-ssreflect/coq-mathcomp-ssreflect.1.7.0/opam

```diff
@@ -2,16 +2,17 @@ opam-version: "1.2"
 name: "coq-mathcomp-ssreflect"
 version: "1.7.0"
 maintainer: "Mathematical Components <mathcomp-dev@sympa.inria.fr>"
+synopsis: "The Mathematical Components library"
 
-homepage: "http://math-comp.github.io/math-comp/"
+homepage: "https://math-comp.github.io/math-comp/"
 bug-reports: "Mathematical Components <mathcomp-dev@sympa.inria.fr>"
 dev-repo: "https://github.com/math-comp/math-comp.git"
 license: "CeCILL-B"
 
-build: [ make "-C" "mathcomp/ssreflect" "-j" "%{jobs}%" ]
-install: [ make "-C" "mathcomp/ssreflect" "install" ]
+build: [ make "-C" "mathcomp/ssreflect" "-C" "mathcomp/ssreflect" "-j" "%{jobs}%" ]
+install: [ make "-C" "mathcomp/ssreflect" "-C" "mathcomp/ssreflect" "install" ]
 remove: [ "sh" "-c" "rm -rf '%{lib}%/coq/user-contrib/mathcomp'" ]
-depends: [ "coq" { ((>= "8.6" & < "8.9~") | (= "dev"))} ]
+depends: [ "coq" { ((>= "8.6" & < "8.10~") | (= "dev"))} ]
```

Comment:
I am not sure I understand why we put an upper version limit here [thus requiring updating when a new Coq is released]

Reply:
I do not know what best practices are... should I remove the upper bound? There is not guarantee a given version of mathcomp will be compatible with future versions of Coq, though... When a new Coq is released and tested against older mathcomp releases, we may bump up the upper bound...

Reply:
Indeed the "upper bound" debate is old an unresolved in the packaging world as far as I can tell.

I think indeed not having upper bounds by default is a common trend in package universes where compatibility "is desired"; adding such constraints only in the case of breakage. This works well for example in C libraries which tend to be ABI-compatible.